Gems/PhysX/Code/Source/System/PhysXSystem.cpp

```diff
@@ -118,12 +118,16 @@ namespace PhysX
         AZStd::string platformName = AZ::GetPlatformName(AZ::g_currentPlatform);
         auto logCategory =
             AZStd::string::format("%.*s-%s", AZ_STRING_ARG(PerformanceLogCategory), platformName.c_str());
+        auto fileExtension =
+            AZStd::string::format("%.*s.json", AZ_STRING_ARG(PerformanceLogCategory));
+        AZStd::to_lower(fileExtension.begin(), fileExtension.end());
         m_performanceCollector = AZStd::make_unique<AZ::Debug::PerformanceCollector>(
-            logCategory,
+            logCategory, 
```

Comment:
Nit: Looks like some extra whitespace got introduced here, not sure if it's just how the diff is being rendered.